agent.py

```diff
@@ -898,8 +895,8 @@ def findNeighborhood(self, newCell=None):
         else:
             newNeighborhood = self.findCellsInRange(newCell)
         neighborhood = []
-        for neighborCell in newNeighborhood:
-            neighbor = neighborCell["cell"].agent
+        for neighborCell in newNeighborhood.keys():
```

Comment:
I'm okay with using `keys()` and `items()` going forward, but there are a lot of instances where I set things up to be more old school (and less Pythonic). As a separate PR (once this is merged), someone should walk through the code and replace instances of loops which can be refactored to use `keys()` or `items()`.